pkg/operator/ceph/object/admin.go

```diff
@@ -214,7 +213,7 @@ func RunAdminCommandNoMultisite(c *Context, expectJSON bool, args ...string) (st
 	var err error
 
 	// If Multus is enabled we proxy all the command to the mgr sidecar
-	if c.CephClusterSpec.Network.IsMultus() {
```

Comment:
Does this was not working?

Hope you have tested the changes by creating the buckets claims and checked every thing works fine....

Reply:
In some parts of the code, object context was not set with clusterspec, we already had the Network info in clusterinfo.  Hence changed this check instead of filling clusterspec to object context